deps/corepack/shims/vcc

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
+
+case `uname` in
+    *CYGWIN*) basedir=`cygpath -w "$basedir"`;;
+esac
+
+if [ -x "$basedir/node" ]; then
+  exec "$basedir/node"  "$basedir/../dist/vcc.js" "$@"
+else 
+  exec node  "$basedir/../dist/vcc.js" "$@"
```

Comment:
What is vcc? How many executables does this PR add?

Reply:
It's not a binary (`vcc.js` isn't executable), it's just [`v8-compile-cache`](https://github.com/zertosh/v8-compile-cache), [bundled in a single file]((https://github.com/nodejs/corepack/blob/main/webpack.config.js#L10)), to help decrease the Node startup time for the binaries spawned by Corepack. This shim you point is an extraneous file that shouldn't be there, I'll see to remove it.

Reply:
ah ok thanks, i guess i misunderstood what the shims are.

which binaries does corepack add to the PATH?

Reply:
`pnpm` / `pnpx` / `yarn` / `yarnpkg`

Reply:
It also adds `corepack` itself

Reply:
so that's 5 initial binaries, with every newly package manager adding potentially 2 more? what's the plan for the future, when that rapidly fails to scale?

Reply:
I don't think the same question was asked of npm when they added `npx`, but generally I wouldn't expect other package managers (should they be written) to be encouraged to have more than a single binary, with supporting more needing a strong justification.

In the case of Yarn specifically, the `yarnpkg` name only exists due to some early conflict in Debian packages. Nowadays 
we only recommend `yarn`, with `yarnpkg` only being in this list for backward compatibility.

Reply:
Sure, but that’s only one package manager :-) there could potentially be many more than one via corepack.